libs/package.json

```diff
@@ -32,9 +32,11 @@
   },
   "dependencies": {
     "@audius/hedgehog": "1.0.12",
+    "@audius/anchor-audius-data": "file:../solana-programs/anchor/audius-data",
     "@certusone/wormhole-sdk": "0.1.1",
     "@ethersproject/solidity": "5.0.5",
     "@improbable-eng/grpc-web-node-http-transport": "0.15.0",
+    "@project-serum/anchor": "0.22.0",
```

Comment:
can upgrade this to match anchor audius data minor version 24

Reply:
done